code/modules/antagonists/clockcult/clock_structures/traps/brass_skewer.dm

```diff
@@ -50,14 +50,20 @@
 	if(density)
 		return
 	var/mob/living/squirrel = locate() in get_turf(src)
+	playsound(src, 'sound/machines/clockcult/brass_skewer.ogg', 75, FALSE)
+	icon_state = "[initial(icon_state)]_extended"
+	density = TRUE //Skewers are one-use only
+	desc = "A vicious brass spike protruding from the ground like a stala[pick("gm", "ct")]ite. It makes you sick to look at." //is stalagmite the ground one? or the ceiling one? who can ever remember?
```

Comment:
the icon state and desc should be part of ``update_icon_state`` and ``update_desc``